app/templates/services/_base_service_page.html

```diff
@@ -78,30 +75,27 @@
           ],
           field_headings_visible=False
         ) %}
-          {% call summary.row(complete=not question.answer_required) %}
+          {% call summary.row() %}
             {{ summary.field_name(question.label) }}
-            {% if question.answer_required %}
-              {% call summary.field() %}
-                {% if framework and framework.status == 'open' %}
-                    <a href="{{ url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id, _anchor=question.id) }}">Answer required</a>
+            {% if question.is_empty %}
+              {% if framework and framework.status == 'open' and section.edit_questions %}
+                {% if question.is_empty %}
+                  {{ summary.link('Answer question', url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id, question_slug=question.slug)) }}
                 {% else %}
-                  Not answered
+                  {{ summary.field("Not answered") }}
                 {% endif %}
-              {% endcall %}
-            {% elif question.is_empty %}
-              {% call summary.field() %}
-                <span class="summary-item-field-answer-required">{{ question.empty_message }}</span>
```

Comment:
Are we not showing these empty messages at all any more?  They were introduced for DOS submissions, and I think Ralph was pretty strong on having them in the DOS summary tables.

It might be worth keeping this in, but ensure we don't show anything if the `empty_message` isn't set (it isn't for G-Cloud submissions).